cdk/domino_cdk/config/s3.py

```diff
@@ -13,30 +13,37 @@ class S3:
     removal_policy_destroy: true/false - Delete bucket when destroying stack. If auto_delete_objects is false,
                                          destroys will fail unless buckets are emptied.
     sse_kms_key_id: XXX - Specific KMS key ID to setup with bucket. Otherwise, defaults to AES256
+    require_encryption: true/false - Require all requests and objects to be encrypted through policies.
+                                     Defaults to true, changing to false does not disable default KMS key encryption of objects.
```

Comment:
Should we maybe just leave this hard on?

Reply:
Yeah this shouldn't be exposed as a config option. It's required for the monitoring / access logs bucket, but it can be an internal flag only.